test/integration/targets/connection_ssh/test_ssh_askpass.yml

```diff
@@ -0,0 +1,57 @@
+- hosts: all
+  tasks:
+    - import_role:
+        role: setup_test_user
+
+    - find:
+        paths: /etc/ssh
+        recurse: true
+        contains: 'PasswordAuthentication'
+      register: sshd_confs
+
+    - lineinfile:
+        path: '{{ item }}'
+        regexp: '^PasswordAuthentication'
+        line: PasswordAuthentication yes
+      loop: '{{ sshd_confs.files|default([{"path": "/etc/ssh/sshd_config"}], true)|map(attribute="path") }}'
+
+    - when: ansible_facts.system == 'Darwin'
+      block:
+        - command: launchctl stop com.openssh.sshd
+        - command: launchctl start com.openssh.sshd
+
+    - service:
+        name: ssh{{ '' if ansible_facts.os_family == 'Debian' else 'd' }}
+        state: restarted
+      when: ansible_facts.system != 'Darwin'
+
+    - command:
+        argv:
+          - ansible
+          - localhost
+          - -m
+          - command
+          - -a
+          - id
+          - -vvv
+          - -e
+          - ansible_connection=ssh
```

Comment:
I'd add `ansible_pipelining=yes` here to speed up the test and reduce output.

Reply:
added